pages/common/gist.md

```diff
@@ -0,0 +1,23 @@
+# gist
+
+> Upload code to https://gist.github.com.
+
+- Upload any number of files:
+
+`gist {{file.txt}} {{file2.txt}}`
+
+- Upload a private file with a description:
+
+`gist -p {{file.txt}} -d {{"A meaningful description"}}`
+
+- Read contents from STDIN with a name:
+
+`{{echo hello world}} | gist -f {{name.rb}}`
+
+- List public and private gists for authed user:
+
+`gist -l`
+
+- Update existing gists with the `GIST_ID` in the URL:
```

Comment:
Oooh good questions. I think the answers should be made clear in the wording here.

Also, a random fact for you. Gists are full git repos, so you can clone & push them as you would any other repos - but you probably knew that already :P